external-dns.tf

```diff
@@ -25,9 +25,9 @@ resource "helm_release" "external-dns" {
   create_namespace = true
 
   name        = "external-dns"
-  repository  = "https://charts.bitnami.com/bitnami"
+  repository  = "https://kubernetes-sigs.github.io/external-dns"
   chart       = "external-dns"
-  version     = "6.18.0"
+  version     = "1.13.1"
```

Comment:
out of interest, does this change work correctly over an already-installed cluster? switching the upstream helm package? or would this change only work on a new instance?

Reply:
It does work yeah on an already-installed cluster, but a one-off in this case and I'd not recommend / risk it in the future. Probably instead switch the release name to ensure it's unique.